### PR TITLE
Add age bar

### PR DESCRIPTION
**Pull request by moos**
Basically inspired by http://www.bertrandplanes.com/pages/LifeClock3.php.

Also:
- Fixes center text alignment on first draw
- Ensures font is loaded
- Use more accurate seconds update using setTimeout
- Added ability to scale all bars

### CODE REVIEW

**Comment by eouia** [comment]
Interesting. 
Can you add a more detailed explanation on README?
The other issue would be, you've added a hand for life age, but is there risk of conflict existing configuration with this new version?

File: MMM-OClock.js [hunk 1]
```diff
@@ -31,8 +32,15 @@ Module.register("MMM-OClock", {
     colorTypeRadiation: ["#333333", "red"], //Don't use #pattern or colorName.
     colorTypeTransform: ["blue", "red"],
     colorTypeHSV: 0.25, //hsv circle start color : 0~1
+    secondsUpdateInterval: 1000,  // msec
```
Comment (moos) [inline MMM-OClock.js]: Forgot to add this in previous PR.

File: MMM-OClock.js [hunk 2]
```diff
@@ -31,8 +32,15 @@ Module.register("MMM-OClock", {
...
+
+    birthYear: false,  // e.g. 1901
+    birthMonth: 0,    // e.g. 1-12
+    lifeExpectancy: MAX_LIFETIME, // default: 85
+    linearLife: false,  // set to true to plot life linearly not logarithmically
 
     handConversionMap: {
+      "age": "n/a",
```
Comment (moos) [inline MMM-OClock.js:47]: Use separate name so not to conflict with existing "year".

**Comment by moos** [comment]
I merged from master before realizing you had also merged the other PR.  Also addressed your comments.  Now using "age" so it doesn't conflict with existing config files.

Here's a screenshot (46, not my true age!):

oclock-age

**Comment by moos** [comment]
OK -- think I'm done.  Hopefully you'll find this useful.

I think the age bar needs some decoration to stand-out from the others.  Maybe an exercise for someone else.

**Comment by moos** [comment]
Oops, I lied.  I use [delightedCrow/WallberryTheme](https://github.com/delightedCrow/WallberryThem) and the OClock looks too contrast-y.  So adding ability to add CSS via config.

**Pull request merged**